linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java

```diff
@@ -180,22 +183,29 @@ public void testBomSnapshot()
   @Test
   public void generateMessageForNewError() {
     Set<SymbolProblem> baselineProblems = ImmutableSet.of(classNotFoundProblem);
+    Path jar = Paths.get("aaa", "ccc-1.2.3.jar");
```

Comment:
This dummy JAR file is renamed from bbb to ccc to distinguish it from [the line 178's dummy data](https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/983/files#diff-36a547c579afc122ddf52d51a6199561L178).